Explorer/Assets/DCL/LOD/Components/LODCacheInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using ECS.StreamableLoading.AssetBundles;
+using Segment.Concurrent;
+using UnityEngine;
+using UnityEngine.Serialization;
+using Utility;
+
+namespace DCL.LOD.Components
+{
+    public struct LODCacheInfo : IDisposable
```

Comment:
let's introduce some encapsulation here to make it clear what can be modified and what can't be: e.g. `LODGroup` and `LODAssets` can be assigned from `ctor` only and thus should be `readonly`